src/content/blog/you-probably-don't-need-eslint-config-prettier-or-eslint-plugin-prettier/index.mdx

```diff
@@ -0,0 +1,212 @@
+---
+description: "Explaining how two popular Prettier integrations for ESLint work, and why I avoid both of them."
+image:
+  alt: Josh smiling on stage and giving a presentation showing the "Type Checking React (v2)" from the linked slides.
+  src: react-miami-me-speaking.jpg
+pubDate: 2023-12-28
+tags: [
+	"biome",
```

Comment:
[Todo] I'd started writing up an explainer about Rust-based linters and type checking performance, but ended up removing it. That'll be a separate blog post. #141.